packages/components/src/index.ts

```diff
@@ -26,7 +27,8 @@ export {
   Bullet,
   Marker,
   Divider,
-  AvatarButton,
+  Avatarbutton,
```

Comment:
```suggestion
  AvatarButton,
```

You might have accidently pulled this PR https://github.com/momentum-design/momentum-design/pull/1075
Please remove this changes and you are good to go.